### PR TITLE
Trn18 payment allocation

### PR DESCRIPTION
**Pull request by KotiGungi-Vienna**
Payment allocation.

### CODE REVIEW

File: VIS/Areas/VIS/Controllers/PaymentAllocationController.cs [hunk 1]
```diff
@@ -201,6 +224,24 @@ public JsonResult GetDocType()
...
+        /// </summary>
+        /// <returns>List of Data Types</returns>
+        public JsonResult GetpayDocType()
+        {
+            Ctx ct = Session["ctx"] as Ctx;
+            PaymentAllocation payments = new PaymentAllocation(ct);
+            return Json(JsonConvert.SerializeObject(payments.GetpayDocType()));
+        }
+
+        public JsonResult GetPaymentType()
```
Comment (Amit-Vienna) [inline VIS/Areas/VIS/Controllers/PaymentAllocationController.cs:238]: do commenting

Comment (KotiGungi-Vienna) [reply VIS/Areas/VIS/Controllers/PaymentAllocationController.cs:258]: Okay sir!

Comment (KotiGungi-Vienna) [reply VIS/Areas/VIS/Controllers/PaymentAllocationController.cs:258]: Payment Allocation, done the Changes in GL Grid.

**Comment by KotiGungi-Vienna** [comment]
Payment allocation.

**Pull request closed without merging**